### DISCUSSION TITLE
spartacus-surface 3D discretisation

### DISCUSSION THREAD

**Discussion by rarygit** [UMEP-dev/UMEP#331] [Q&A]
Hi Ting,

Looking at the 3D vegetated urban surface type schematic for spartacus-surface,
https://github.com/ecmwf/spartacus-surface/blob/master/doc/surface_type_schematic.pdf

Does this mean that we could obtain outputs in SUEWS at the microscale level using spartacus-surface?  
Output for individual or selected sub-regions, individual buildings or trees?

Or is it only local scale output?

**Comment by sunt05** [comment]
Hi @rarygit, sorry about missing this thread for such a long time. 

Yes, we can get the output at a very detailed level; however, as the coupling is in progress, things are subject to change.
Anyway, we are willing to hear users' voice and implement as many as expected and possible.

**Comment by rarygit** [comment]
Hi Ting, thanks for replying!

Detailed output at the microscale level would be very useful, in conjunction with existing refinements in the SUEWS model:

 - Soil moisture can either be provided using observational data in the met forcing file and providing some metadata information here (OBS columns), or modelled by SUEWS
https://suews-docs.readthedocs.io/en/latest/input_files/SUEWS_SiteInfo/SUEWS_Soil.html?highlight=observed%20soil#suews-soil-txt

Observed soil moisture values can incorporate both runon and irrigation inputs.

At the microscale, it would be useful (or necessary) to differentiate observed soil moisture values under specific combinations of SUEWS surfaces and land cover types. For example, deciduous trees planted above pervious paving landcover types. Alternatively trees planted over grassed areas.

For this combination, the SUEWS_Soil.txt could represent tree trench pits, with hourly soil moisture provided using observational data in the met forcing file. This would require a met file with specific column names for separate areas having different soil moisture observations. 

Without getting into too much detail here, I would like to use observed soil moisture values to drive microscale subcanopy convective fluxes (latent energy and sensible heat). The objective is to model human thermal comfort levels.
